test/e2e/kube-ovn/subnet/subnet.go

```diff
@@ -750,20 +750,43 @@ var _ = framework.Describe("[group:subnet]", func() {
 		err = deployment.WaitForDeploymentComplete(cs, deploy)
 		framework.ExpectNoError(err, "deployment failed to complete")
 
-		checkFunc := func(usingIPRange, availableIPRange, startIP, lastIP string, count int64) {
+		checkFunc := func(usingIPRange, availableIPRange, startIP, lastIP string, count int64, isFrameworkCheck bool) bool {
 			usingIPEnd := util.BigInt2Ip(big.NewInt(0).Add(util.Ip2BigInt(startIP), big.NewInt(count-1)))
 			availableIPStart := util.BigInt2Ip(big.NewInt(0).Add(util.Ip2BigInt(usingIPEnd), big.NewInt(1)))
-			framework.ExpectEqual(usingIPRange, fmt.Sprintf("%s-%s", startIP, usingIPEnd))
-			framework.ExpectEqual(availableIPRange, fmt.Sprintf("%s-%s", availableIPStart, lastIP))
+			if isFrameworkCheck {
+				framework.ExpectEqual(usingIPRange, fmt.Sprintf("%s-%s", startIP, usingIPEnd))
+				framework.ExpectEqual(availableIPRange, fmt.Sprintf("%s-%s", availableIPStart, lastIP))
+				return true
+			} else {
+				return usingIPRange == fmt.Sprintf("%s-%s", startIP, usingIPEnd) && availableIPRange == fmt.Sprintf("%s-%s", availableIPStart, lastIP)
+			}
+		}
+
+		checkTimes := 0
+		isSuccess := false
+		maxRetryTimes := 30
+		for {
+			time.Sleep(1 * time.Second)
+			if checkTimes > maxRetryTimes || isSuccess {
+				break
+			}
```

Comment:
We can replace the loop with `wait.PollImmediate`.

Reply:
done